### ISSUE TITLE
[Q]Question about wandb logging memory

### ISSUE THREAD

**Issue by necrophagists** [wandb/wandb#6913]
Hi, I've been using it recently and noticed that wandb logs memory information even when trainer's args.skip_memory_tracker=True, how can I turn off this setting in wandb?

**Comment by fmamberti-wandb** [comment]
Hi @necrophagists - thank you for reaching out. 

You can disable logging system information using `wandb.init(settings=wandb.Settings(_disable_stats=True))` however, this will disable logging all system metrics, not only memory information.

If you don't mind providing more details on your use case for disabling logging memory information and the impact this feature would have on your workflow, we can capture the feedback and pass it to our product team for review.

**Comment by necrophagists** [comment]
> Hi @necrophagists - thank you for reaching out.
> 
> You can disable logging system information using `wandb.init(settings=wandb.Settings(_disable_stats=True))` however, this will disable logging all system metrics, not only memory information.
> 
> If you don't mind providing more details on your use case for disabling logging memory information and the impact this feature would have on your workflow, we can capture the feedback and pass it to our product team for review.

Thank you very much for your reply. I found this problem by executing LLAVA's training script by adding `report_to ='wandb' ` to `training_args` in the transformers library.
Then I looked at the trainer's source code and found that it doesn't track memory and such by default (`args.skip_memory_tracker=True`), and it doesn't change that after setting `report_to = 'wandb'`. I guess the possible reason for this is that wandb has its own set of monitoring programs?

**Comment by fmamberti-wandb** [comment]
> I guess the possible reason for this is that wandb has its own set of monitoring programs?

Yes, this is correct, `wandb` has its own monitoring of system resources. It can be turned off as mentioned above - however, it is not currently possible to stop reporting on memory (or other individual resources) only. Let me know if you have any further questions.
